pkg/metautil/model.go

```diff
@@ -0,0 +1,56 @@
+package metautil
+
+import (
+	"time"
+)
+
+type Model struct {
+	ID        uint      `gorm:"primaryKey"`
+	CreatedAt time.Time // NOTICE: autoupdate inside the orm lib, but not sql backend
+	UpdatedAt time.Time
+}
+
+type ProjectInfo struct {
+	Model
+	ProjectID   string `gorm:"type:char(64) not null;uniqueIndex:uidx_id"`
+	ProjectName string `gorm:"type:varchar(64) not null"`
+}
+
+type ProjectOperation struct {
+	ID        uint      `gorm:"primaryKey;auto_increment"`
+	ProjectID string    `gorm:"type:char(64) not null;index:idx_op"`
+	Operation string    `gorm:"type:varchar(16) not null"`
+	JobID     string    `gorm:"type:char(64) not null"`
+	CreatedAt time.Time `gorm:"autoCreateTime;index:idx_op"`
+}
+
+type JobInfo struct {
+	Model
+	ProjectID string `gorm:"type:char(64) not null;index:idx_st,priority:1"`
+	JobID     string `gorm:"type:char(64) not null;uniqueIndex:uidx_id"`
```

Comment:
Yes, it's the uuid the framework used. 
The precise length is 35 bytes, may be 36 is the most proper num.
Since the uuid is always the same length, so I prefer char instead of varchar.

Reply:
So why choosing 64 instead of 36

Reply:
Preserve some bytes in case of some change future.
But I will change to 36 for now.